src/parse/init.go

```diff
@@ -23,6 +23,12 @@ type aspParser struct {
 
 // newAspParser creates and returns a new asp.Parser.
 func newAspParser(state *core.BuildState) *asp.Parser {
```

Comment:
This function isn't useful? it's just a clone of GetParserWithBuiltins?

Reply:
oops, I moved things around then forgot to move it back